renovate.json

```diff
@@ -1,6 +0,0 @@
-{
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "local>balena-os/renovate-config"
```

Comment:
We want to change this to `"github>balena-io/renovate-config"` not delete the file